main.go

```diff
@@ -168,7 +168,7 @@ func checkBuf(path string, fileBytes []byte) (int, *bytes.Buffer, error) {
 		}
 		// If goimports made any change, count that as a diff so the file
 		// can be overwritten at the end.
-		if bytes.Compare(fileBytes, newFileBytes) != 0 {
+		if bytes.Compare(fileBytes, newFileBytes) != 0 && *printDiff {
```

Comment:
maybe good to put *printDiff first to short circuit the work of bytes.Compare if it's off.

Reply:
Done